dependency-suppression/cve-suppressed.xml

```diff
@@ -77,29 +77,29 @@
         <notes><![CDATA[
    file name: spring-pulsar-spring-cloud-stream-binder-0.2.0
    ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.springframework\.pulsar/spring\-pulsar\-spring\-cloud\-stream\-binder@.*$</packageUrl>
+        <packageUrl regex="true">^pkg:maven\/org\.springframework\.pulsar\/spring\-pulsar\-spring\-cloud\-stream\-binder@.*$</packageUrl>
         <cve>CVE-2022-31678</cve>
         <cve>CVE-2022-31701</cve>
     </suppress>
     <suppress>
         <notes><![CDATA[
    file name: jackson-databind-*.jar
    ]]></notes>
-        <packageUrl regex="true">^pkg:maven/com.fasterxml.jackson.core/jackson-databind@.*$</packageUrl>
+        <packageUrl regex="true">^pkg:maven\/com.fasterxml.jackson.core\/jackson-databind@.*$</packageUrl>
         <cve>CVE-2023-35116</cve>
     </suppress>
     <suppress>
         <notes><![CDATA[
    file name: protoc-gen-grpc-kotlin-1.3.0-jdk8.jar (shaded: com.google.guava:guava:31.0.1-android)
    ]]></notes>
-        <filePath regex="true">.*\/protoc-gen-grpc-kotlin-1.3.0-jdk8.jar\/.*\/com.google.guava/guava/pom.xml</filePath>
+        <filePath regex="true">.*\/protoc-gen-grpc-kotlin-1.3.0-jdk8.jar\/.*\/com.google.guava\/guava\/pom.xml</filePath>
         <cve>CVE-2023-2976</cve>
     </suppress>
     <suppress until="2023-10-01">
         <notes><![CDATA[
    file name: quartz-2.3.2.jar
    ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.quartz\-scheduler/quartz@.*$</packageUrl>
+        <packageUrl regex="true">^pkg:maven\/org\.quartz\-scheduler\/quartz@.*$</packageUrl>
```

Comment:
this seems a very schedule kind of exception, shouldn't we have this in just that repo?